Include/cpython/pyatomic_msc.h

```diff
@@ -912,6 +912,30 @@ _Py_atomic_store_ptr_release(void *obj, void *value)
 #endif
 }
 
+static inline void
+_Py_atomic_store_int_release(int *obj, int value)
+{
+#if defined(_M_X64) || defined(_M_IX86)
+    *(int volatile *)obj = value;
```

Comment:
@colesbury 
Basically, int volatile * and volatile int* has the same meaning.
So I just follow the convention from `_Py_atomic_load_ptr_acquire`